command-line/command-line-advanced.md

```diff
@@ -10,7 +10,10 @@ About 1 hour
 
 ### Motivation
 
-Teach people some handier tools for advanced file searching and processing
+Teach people some handier tools for advanced file searching and processing.The Command line makes  work easier for developers.
+
```

Comment:
So although you need to cite sources, this section is supposed to be very short and, well, motivational.  Could you please summarize in a few lines or bullet points what he’s saying in the article, and then cite?